pyserverforindy/grpc_server/anoncreds_service.py

```diff
@@ -449,6 +545,10 @@ class AnoncredsServiceServicer(identitylayer_pb2_grpc.AnoncredsServiceServicer):
                                                                 rev_reg_def_json, rev_reg_delta_json, timestamp,
                                                                 cred_rev_id)
             return identitylayer_pb2.UpdateRevocationStateResponse(RevReg=resp[0], Witness=resp[0], Timestamp=resp[0])
+        except IndyError as e:
+            logger.error("Indy Exception Occurred @ UpdateRevocationState ------")
+            logger.error(e.message)
+            return identitylayer_pb2.CreateRevocationStateResponse()
         except Exception as e:
             logger.error("Exception Occurred @UpdateRevocationState ------")
```

Comment:
No major changes in annoncreds_service. just InnyError handler for future changes